pkg/driver/helmdriver.go

```diff
@@ -243,3 +246,43 @@ func (d *helmDriver) IsConfigChanged(_ context.Context, name string, values map[
 
 	return !reflect.DeepEqual(values, rel.Config), nil
 }
+
```

Comment:
Do you need a note that these are just copy/pasta and can be removed when helm releases this stuff?

Reply:
With the current PR, this change is needed regardless of whether helm releases the fix because these changes are done on the CLI and not the main path so mimicking this in our repo is required.